documentation/basic-tutorial.md

```diff
@@ -0,0 +1,59 @@
+# Basic Tutorial
+
+In this tutorial, we'll create a BTC/USD token collateralized with an ERC-20 token (a stand in for DAI) via the
+UMA token builder. We will spin up the system locally on Ganache, push a price for BTC/USD, and create a token.
+
+## Prerequisites
+
+All of our tutorials require that you complete the steps in `prerequisites.md`.
+
+## Deploy smart contracts to Ganache
+
+Run all commands in this section from the `core/` directory.
+
+1. Start the Ganache UI.
+1. Switch to a test identifier config:
+```
+mv core/config/identifiersTest.json core/config/identifiers.json
+```
+1. Deploy smart contracts to the locally running Ganache instance via:
+```
+$(npm bin)/truffle migrate --reset --network test
+```
+
+After this step, the smart contracts are deployed and can be interacted with.
+
+## Pushing prices (manually)
+
+Run all commands in this section from the `core/` directory.
+
+In production, we can run a job on a cloud provider that fetches prices from a data API and pushes them to the
+blockchain. An example of such a script is `core/scripts/PublishPrices.js`.
+
+Locally, however, let's skip the step of setting up a data feed and manually push a price for `BTCUSD` via:
+
+```
+$(npm bin)/truffle exec scripts/ManualPublishPriceFeed.js --identifier BTCUSD --price <price> --time <time>
+```
+
+where `<price>` is the current price of Bitcoin in USD and `<time>` is the current UNIX timestamp in seconds.
+
+For example, if the price of Bitcoin were `$8,923` at 10/21/2019 15:40 EDT, run:
+```
+$(npm bin)/truffle exec scripts/ManualPublishPriceFeed.js --identifier BTCUSD --price 8293 --time 1571686800
```

Comment:
I don't have an opinion here  - just curious if why you chose to use this script rather than the one that just pulls a price from coinbase and uploads it with no user inputs? Just to keep it low level so the user could understand exactly what they're doing or something else?

Reply:
We could go with that too, I just thought that the PublishPrices.js script is very complicated and has tons of extra features that don't matter to someone just spinning up the basics. Thoughts?

Reply:
SGTM